samples/matter/light_bulb/README.rst

```diff
@@ -301,7 +283,14 @@ If you are new to Matter, the recommended approach is to use :ref:`CHIP Tool for
 
 Before starting the commissioning procedure, the device must be made discoverable over Bluetooth LE.
 The device becomes discoverable automatically upon the device startup, but only for a predefined period of time (15 minutes by default).
-If the Bluetooth LE advertising times out, press **Button 4** to re-enable it.
+If the Bluetooth LE advertising times out:
+
+   * On nRF52840 DK, nRF5340 DK and nRF21540 DK:
+     * press **Button 4**
+   * On nRF7002 DK:
+     * press **Button 2** for at least three seconds
```

Comment:
```suggestion
   * On nRF52840 DK, nRF5340 DK and nRF21540 DK:
     
     * Press **Button 4**.
   
   * On nRF7002 DK:
     
     * Press **Button 2** for at least three seconds.
```

Reply:
This part is not needed in other sample docs?

Reply:
I think no because light bulb and template are only apps that use auto start BLE advertising feature and we do not support re-enabling it with button for the latter.